.github/workflows/release.latest.yml

```diff
@@ -21,3 +21,4 @@ jobs:
           file: cmd/proxy/Dockerfile
           build-args: VERSION=${{github.sha}}
           tags: gomods/athens:latest
+          push: true
```

Comment:
We are not pushing a tagged image here, only latest, which no one uses in production.

Reply:
Right, will update later. Idk how it was working in drone though 🤔. (maybe I missed a step while migrating)

Reply:
https://github.com/gomods/athens/commit/3f26845cffd789688277c8a79d1cc11159a2c689#diff-b54b39f1afced2465e1f3641db9d5bbf4f3a7fcf890996dfedd3c197bcb7f8c7L123

Reply:
Updated. Thanks for pointing out!